### ISSUE TITLE
Users can download file by FileUploadField when they don't have permissions to read FileDescriptor

### ISSUE THREAD

**Issue by haulmont-git** [cuba-platform/cuba#621]
*Prerequisites*
```
cuba.entityAttributePermissionChecking = false (default behavior) 
```

Scenario to reproduce:
1. Import roles (Role accepts restrictions for FileDescriptor (all operations denied))
1. Create user with name Test. Assign the role "TestRole"
1. Create entity A and upload file for it
1. Login by the user (Test).
1. Open screen A.browser
1. Select entity and open the edit screen
1. Try to download file

AR: User can download file
ER: User can not download file

Documentation:
```
cuba.entityAttributePermissionChecking

If set to false, turns off checking entity attribute permissions on Middleware.
Default value: false
Stored in the database.
Used in the Middleware block.
```

:link: Related links: {"relates to:": "https://youtrack.haulmont.com/issue/PL-9935"}

---
Original issue: https://youtrack.haulmont.com/issue/PL-10013

**Comment by dtsaryov** [comment]
@GlebDurygin check if fixed in #622

**Comment by GlebDurygin** [comment]
duplicates #622
